### PR TITLE
fix: persist kubernetes config

### PR DESCRIPTION
**Pull request by hftsin**
## TL; DR

This PR fixes an issue that the traefik component is still launched even it is disabled by default in the `colima.yaml` config.

## Issue Description

On starting colima with kubernetes disabled, the kubernetes config is not persisted to the filesystem of the guest OS. Later we start the kubernetes and it failed to load the kubernetes config. As a side effect, it still launches the traefik component even it is disabled by default.

## Solution

- Persist the kubernetes config to the filesystem of guest OS on starting colima
~~- Add traefik to the default disabled component list~~

### CODE REVIEW

File: environment/container/kubernetes/k3s.go [hunk 1]
```diff
@@ -122,6 +122,9 @@ func installK3sCluster(
 		"--resolv-conf", "/etc/resolv.conf",
 	}
 
+	// disable traefik defaults
+	args = append(args, "--disable", "traefik")
```
Comment (abiosoft) [inline environment/container/kubernetes/k3s.go:126]: this is going to make traefik always disabled

Comment (hftsin) [reply environment/container/kubernetes/k3s.go:126]: yes, this is going to disable traefik always. I revert this commit.

**Review by abiosoft — APPROVED** [review]

**Comment by hftsin** [comment]
I want to re-trigger the workflow by force-push since the previous one failed. https://github.com/abiosoft/colima/actions/runs/3686095318

**Comment by hftsin** [comment]
I am so sorry that there is still one failing CI check. I have to bother you to trigger it again. @abiosoft

**Comment by abiosoft** [comment]
> I am so sorry that there is still one failing CI check. I have to bother you to trigger it again. @abiosoft

no worries, it's fine.

**Pull request merged**